lib/spandex_datadog/sampling_strategies/keep_all.ex

```diff
@@ -0,0 +1,8 @@
+defmodule SpandexDatadog.SamplingStrategies.KeepAll do
```

Comment:
Shouldn't this strategy be part of `spandex`? It seems pretty generic.

Reply:
well
1. spandex only support datadog anyway at this point
2. the precise `priority` value can be specific to the adapter
3. The thing is so simple and basic that I don't see anything wrong with duplicating it if one day someone makes a new adapter
4. I didn't think decoupling the generic part of the sampling algorithm from the adapter-specific parts would make anything any more clear. I'd rather have it all in one piece